data/BEPRO/data.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Bepro",
```

Comment:
Validation is failing because this name doesn't match the name in the ethereum token contract (BetProtocolToken).  You can specify a name override in the Ethereum section below, or by redeploying one of the contracts to have a matching name.